tests/src/test/java/alluxio/client/fuse/FuseFileSystemIntegrationTest.java

```diff
@@ -181,7 +181,7 @@ public void cp() throws Exception {
   }
 
   @Test
-  public void ddAndRm() throws Exception {
+  public void ddAndDu() throws Exception {
```

Comment:
This still does the `rm` portion of the test

Reply:
Changed to `ddDuAndRm`, thanks!